cypress/integration/raise-repair/form.spec.js

```diff
@@ -55,6 +56,14 @@ describe('Raise repair form', () => {
       ]
     )
 
+    // Property contact information table
+    cy.get('.govuk-table__caption').contains('Contacts')
+    cy.get('.govuk-table__row .govuk-table__cell').contains('Mark Gardner')
+    cy.get('.govuk-table__row .govuk-table__cell').contains('00000111111')
+    cy.get('.govuk-table__row .govuk-table__cell').contains('00000222222')
+    cy.get('.govuk-table__row .govuk-table__cell').contains('Luam Berhane')
+    cy.get('.govuk-table__row .govuk-table__cell').contains('00000666666')
```

Comment:
Is it possible to add a data-attribute to identify the rows exactly? Maybe a bit overkill but I find it useful when doing tests on multiple records to make sure i'm targeting the correct index/row etc

Reply:
Thanks. I definitely agree that these should be more focused. I was interested to read about the Cypress recommendation to use data attributes just for testing.

For this case I think I'm mostly interested in the numbered row in the table and I hoped to just rely on the DOM ordering. I've chosen to go for using `cy.eq()` to scope the query to the first and then the second row. I feel a little resistant to adding data attributes to our markup if they don't support the functioning or presentation of the app, but I can also see how that would be super useful for testing too especially as we're using lots of generic classes.

I'm going to go ahead with this for now as it is an improvement hopefully, but happy and interested to talk about this further.

```
    cy.get('.govuk-table')
      .contains('Contacts')
      .parent()     // grab the parent table for scoping the following
      .within(() => {
        cy.get('tbody>tr').eq(0).contains('Mark Gardner')   // first row
        cy.get('tbody>tr').eq(0).contains('00000111111')
        cy.get('tbody>tr').eq(0).contains('00000222222')

        cy.get('tbody>tr').eq(1).contains('Luam Berhane')  // second row
        cy.get('tbody>tr').eq(1).contains('00000666666')
      })
```